UnityProject/Assets/Scripts/Core/Lifecycle/PlayerSpawn.cs

```diff
@@ -310,10 +308,12 @@ private static void ApplyNewSpawnRoleToBody( GameObject body, Occupation request
 
 
 		//Character attributes
-		var name = requestedOccupation.JobType != JobType.AI ? character.Name : character.AiName;
+		var name = requestedOccupation.OrNull()?.JobType != JobType.AI ? character.Name : character.AiName;
 		body.name = name;
 
 
+		Logger.LogError(JsonConvert.SerializeObject(character));
```

Comment:
why error?

Reply:
likely a debug log used while developing and should be removed

Reply:
Yeah will remove